src/kaggle_benchmarks/utils.py

```diff
@@ -40,18 +41,33 @@ def default(self, obj):
         return repr(obj)
 
 
-def build_httpx_client(cache_subdirectory: str = ".") -> httpx.Client:
+class UnconditionalCacheTransport(httpx.HTTPTransport):
+    def handle_request(self, request):
+        response = super().handle_request(request)
+
+        # Force the header to be cacheable for 1 week
+        # This overrides whatever the server said (e.g. no-store)
+        response.headers["Cache-Control"] = "public, max-age=604800"
```

Comment:
nit: shall we make a `cache_duration` parameter just in case and default to a week? I guess ideally we want it to be synced with server side caching.